proto/ibc/lightclients/wasm/v1/tx.proto

```diff
@@ -18,10 +19,12 @@ message MsgStoreCode {
   option (cosmos.msg.v1.signer) = "signer";
 
   string signer = 1;
-  bytes  code   = 2;
+  // WASMByteCode can be raw or gzip compressed
+  bytes wasm_byte_code = 2 [(gogoproto.customname) = "WASMByteCode"];
```

Comment:
See [here](https://github.com/CosmWasm/wasmd/blob/52a7a6ad2c00270867e1979eed1abb48c9ce04df/proto/cosmwasm/wasm/v1/tx.proto#L73).

Reply:
nit: I know these replicate the wasmd protos but for our protodocs we usually use all lowercase